modules/administration/pages/kubernetes.adoc

```diff
@@ -1,175 +1,173 @@
-* ``Runtime``: Overall status of the running instances of the image throughout the registered clusters.
-The status can be one of the following:
-** _All instances are consistent with SUSE Manager_: All the running instances are running the same build of the image as tracked by {productname}.
-** _Outdated instances found_: Some of the instances are running an older build of the image.
-A redeploy of the image into the pod may be required.
-** _No information_: The checksum of the instance image does not match the image data contained in {productname}.
-A redeploy of the image into the pod may be required.
-* ``Instances``: Number of instances running this image across all the clusters registered in {productname}.
-A breakdown of numbers can be seen by clicking the pop-up icon next to the number.
+The image list table contains three columns:
 
+* [guimenu]``Revision``:
++
+An artificial sequence number that increments on every rebuild for images built by {productanme}, or on every import for externally built images.
+* [guimenu]``Runtime``:
++
+Overall status of the running instances for each image in registered clusters.
+* [guimenu]``Instances``:
++
+Number of instances running this image across all the clusters registered in {productname}.
+You can see a breakdown of numbers by clicking the pop-up icon next to the number.
 
+The [guimenu]``Runtime`` column displays one of these status messages:
 
-[[at.build.an.image.for.deployment.in.kubernetes]]
-== Build an image for deployment in Kubernetes
+* ``All instances are consistent with SUSE Manager``:
++
+All the running instances are running the same build of the image as tracked by {productname}.
+* ``Outdated instances found``:
++
+Some of the instances are running an older build of the image.
+You might need to redeploy the image into the pod.
+* ``No information``:
++
+The checksum of the instance image does not match the image data contained in {productname}.
+You might need to redeploy the image into the pod.
 
 
-The following steps will help you build an image for deployment in Kubernetes.
 
-. Under menu:Images[Stores], create an image store.
-. In menu:Images[Profiles], create an image profile (with a dockerfile that is suitable to deploy to Kubernetes).
-. Under menu:Images[Build], build an image with the new profile and wait for the build to finish.
-. Deploy the image into one of the registered Kubernetes clusters using ``kubectl``.
+// This procedure needs help. LKB 2019-10-03
+.Procedure: Building an Image
+. In the {productname} {webui}, navigate to menu:Images[Stores].
+. Click btn:[Create] to create an image store.
+. Navigate to menu:Images[Profiles].
+. Click btn:[Create] to create an image profile.
+You will need to use a dockerfile that is suitable to deploy to {k8s}.
+. Navigate to menu:Images[Build] to build an image with the new profile.
+. Use the [command]``kubectl`` tool to deploy the image into one of the registered {k8s} clusters.
```

Comment:
How they actually deploy the images into the cluster is beyond our scope. `kubectl` is just one way of doing it.